### ISSUE TITLE
calculate partial gradient

### ISSUE THREAD

**Issue by xuzhuoran0106** [tensorflow/tensorflow#6395]
Hi,

For example, in such a multi-task network.

A=net(input)
o1=softmax(A)
B=fullconnect(A)
o2=softmax(B)

Now, the gradient of  'A' from both 'o1' and 'B'.
But, I only want the gradient of  'A' from  'o1'

Is there a way to do this in tensorflow?

Thanks.

**Comment by andydavis1** [comment]
This is a question better suited for StackOverflow, which we also monitor. Please ask it there and tag it with the `tensorflow` tag.
